src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PDFMerger.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.slc.services.steps.getpdf;
+
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import uk.gov.hmcts.reform.slc.services.servicebus.exceptions.PDFMergeException;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.pdfbox.io.MemoryUsageSetting.setupMainMemoryOnly;
+
+public final class PDFMerger {
```

Comment:
We already have `PdfCreator`, `PdfDoc`, `FtpAvailabilityChecker`, `FtpUploader` etc. Let's stick to existing convention (and hmcts checkstyl rules).

Reply:
`PDF` is an abbreviation so it should be `PDF` and not `Pdf`.Happy to discuss about this and not sure about who has finalised hmcts checkstyle rules regarding abbreviation :).Anyways to keep it consistent for now I will update it to `Pdf`